dask/config.py

```diff
@@ -0,0 +1,194 @@
+from __future__ import print_function, division, absolute_import
+
+from contextlib import contextmanager
+import os
+import sys
+
+from .compatibility import FileExistsError
+
+
+config_paths = [
+    '/etc/dask',
+    os.path.join(sys.prefix, 'etc', 'dask'),
+    os.path.join(os.path.expanduser('~'), '.config', 'dask'),
+    os.path.join(os.path.expanduser('~'), '.dask')
+]
+
+if 'DASK_CONFIG' in os.environ:
+    config_paths.append(os.environ['DASK_CONFIG'])
```

Comment:
Should be override, i.e., in position 0? 

-edit-
I see below that later wins. Perhaps a comment here, then. Is it a surprise when providing config explicitly like this, that other values are also read, if not overridden?

Reply:
I get the sense that it's nicer to stack things.  This allows, for example, administrators to specify certain low-level configuration values while also letting users add their own.  I *think* that this is the best approach, but would welcome alternatives.